### DISCUSSION TITLE
Installing ComfyUI-Manager on versions of ComfyUI running in a conda environment on Windows

### DISCUSSION THREAD

**Discussion by greevedogg** [Comfy-Org/ComfyUI-Manager#273] [Q&A]
I am currently running ComfyUI from miniconda in Windows, due to having an AMD GPU. Is there a way to install the ComfyUI-Manager?

**Comment by ltdrdata** [comment]
Just `git clone https://github.com/ltdrdata/ComfyUI-Manager` in ComfyUI/custom_nodes

**Reply by greevedogg** [reply]
Thank you!
